core/src/ast_builder/expr/function.rs

```diff
@@ -379,12 +397,27 @@ pub fn substr<V: Into<ExprNode>>(expr: V, start: V, count: Option<V>) -> ExprNod
     }))
 }
 
+pub fn div<V: Into<ExprNode>>(dividend: V, divisor: V) -> ExprNode {
```

Comment:
I like the decision to have name `modulo` rather than `mod`, how about applying this also for `div`?
As a separate function to be exported, `divide` would be explicit to understand.